phial/wrappers.py

```diff
@@ -195,12 +197,16 @@ def __init__(self,
                  original_ts: Optional[str] = None,
                  attachments: Optional[Union[List[MessageAttachment],
                                        List[MessageAttachmentJson]]] = None,
-                 reaction: Optional[str] = None) -> None:
+                 reaction: Optional[str] = None,
+                 ephemeral: Optional[bool] = False,
```

Comment:
This should just have the type `bool`. `Optional[bool]` suggests `None` is allowed to be passed in which is not the case